### DISCUSSION TITLE
Easiest way to convert asciidoc to qmd?

### DISCUSSION THREAD

**Discussion by cod3licious** [quarto-dev/quarto-cli#11615] [Q&A] [answered]
### Description

I have an existing asciidoc book and would like to switch over to quarto, including converting all the `.adoc` files into `.qmd` files, since I prefer the qmd syntax. Is there an easy way to (at least approximately) convert from one format into the other?

**Comment by mcanouil** [comment]
Use Pandoc to go from Asciidoc to markdown then manual editing for Quarto specific features.

https://pandoc.org

You can use the embedded version of Pandoc via `quarto pandoc ...`.

**Reply by cod3licious** [reply]
Unfortunately, for some weird reason pandoc only likes asciidoc as an output format, not input format, so I get the error `Unknown input format asciidoc` 😔

**Reply by mcanouil** [reply]
Oh, indeed I did not see the arrow was only in one direction.
That's unfortunate.

**Comment by cod3licious** [comment, marked as answer]
Well, far from perfect, but this command seems to be a good start:
```
asciidoctor -b docbook -a leveloffset=+1 -o - README.adoc | \
pandoc --wrap=preserve -t markdown_strict -f docbook - > README.md
```
(see also [here](https://github.com/asciidoctor/asciidoctor/issues/1907))
